content/backmatter.tex

```diff
@@ -334,6 +334,7 @@ \section{Overview}\label{subsec:dep_overview}
     & \shortstack{\textbf{Last Version Supported}}
     & \textbf{Replaced By} \\
     \hline
+    Header Directory: \hyperref[subsec:dep_rationale:mpp]{\HEADER{mpp}} & 1.1 & Current & (none) \\ \hline
```

Comment:
I went with "Header Directory" since it seemed more widely applicable and left room for interpretation of all headers (not just `shmem.h` and `shmem.fh`). It also made it easier to hyperref-link it. I don't mind either, but this seemed cleaner if less precise.

As for the extension headers, I found that to be ambiguous. The OpenSHMEM 1.0 compatibility note where the deprecation rationale was taken from states "these header files", but there are no references to extension headers in 1.0 and `shmemx.h` was not introduced until 1.1. It was assumed it was calling out only `shmem.h`. I kept the original semantics for "these header files" in the rationale, but thinking on it now, I should include `shmemx.h` into the rationale's code blocks.